corehq/apps/domain/calculations.py

```diff
@@ -342,7 +358,7 @@ def calced_props(domain_obj, id, all_stats):
     return {
         "_id": id,
         "cp_n_web_users": int(all_stats["web_users"].get(dom, 0)),
-        "cp_n_active_cc_users": int(CALC_FNS["mobile_users"](dom)),
+        "cp_n_active_cc_users": int(CALC_FNS["active_mobile_users"](dom)),
```

Comment:
Is the new metric not added here because of the performance concern?

Am I correct in saying that until it's added here it won't be available for the SF integration?

Reply:
Yes, that is correct. I was asked to only add it to the SF integration once the performance could be determined. 
Since for large domains this is only taking a few seconds:
- I can probably remove the datadog metric
- I can add the new calculation to the API
Does that sound right to you? 
Thanks for running the performance test!

Reply:
OK, I've made those changes now.
Thanks!